From 66b3c5d711b3321ce43411d6c1f0e6dd2cace113 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Mon, 8 Aug 2022 10:04:58 -0700
Subject: [PATCH] HACK: drm/amd/display: Increase frame_warn_flag value

For x86_64 allmodconfig, a few of the functions in these files are over
this limit, which breaks the build with -Werror.

While we figure out how to fix these properly, increase the limit on
these files to 2500 so that these instances are quiet and no egregious
violations get added in the meantime.

Link: https://github.com/ClangBuiltLinux/linux/issues/1681
---
 drivers/gpu/drm/amd/display/dc/dml/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dml/Makefile b/drivers/gpu/drm/amd/display/dc/dml/Makefile
index 359f6e9a1da0..8030b642cf3c 100644
--- a/drivers/gpu/drm/amd/display/dc/dml/Makefile
+++ b/drivers/gpu/drm/amd/display/dc/dml/Makefile
@@ -51,7 +51,7 @@ endif
 endif
 
 ifneq ($(CONFIG_FRAME_WARN),0)
-frame_warn_flag := -Wframe-larger-than=2048
+frame_warn_flag := -Wframe-larger-than=2500
 endif
 
 CFLAGS_$(AMDDALPATH)/dc/dml/display_mode_lib.o := $(dml_ccflags)
-- 
2.37.1

